featurizers/fingerprints.py

```diff
@@ -386,15 +391,15 @@ def _calculate_fingerprint(
                 },
             )
 
-            # Set a property for each fingerprint, to be able to obtain energy later
-            for i in range(len(fps)):
-                fps[i].set_prop("Energy", energies[i])
-
-                if self.is_folded:
-                    fps[i] = fps[i].fold(self.fold_bits)
+            energies = values[2]
+            fp = fps[np.argmin(energies)]
```

Comment:
Make this an argument to the constructor, e.g. `aggregation_type`, with default value `min_energy` or something like that. This should be expanded in the future, after all.

Reply:
Added an "if" statement for that. The "else" choses the first fingerprint in the list as a placeholder